src/scripts/main.js

```diff
@@ -1,5 +1,34 @@
 'use strict';
 
 document.addEventListener('click', e => {
-  // write code here
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
```

Comment:
move these variables outside the addEventListener (now on every click js would search in DOM and create the same vars again and again)

Reply:
Done!